smart-web-server/src/main/resources/api/parameters/cmdlets/CmdletSort.yaml

```diff
@@ -0,0 +1,14 @@
+title: CmdletSort
+type: string
+description: Sort field names prefixed with '-' for descending order
+enum:
+  - id
+  - ruleId
+  - state
+  - generate_time
+  - state_changed_time
```

Comment:
Consider using `camelCase` for naming variables to align with Java naming conventions. If necessary, implement additional mapping to `snake_case` in the DAO layer

Reply:
Actually, these are not variables but enum constants. They will only be used in the query parameters and codegen will generate enums constants with Java conventions-compliant names. I accidentally didn't replace `ruleId` with `rule_id`. 

However, I replaced `snake_case` names with `camelCase` ones for API uniformity.